plugin/tooling.py

```diff
@@ -535,3 +535,18 @@ def on_transport_close(self, exit_code: int, exception: Optional[Exception]) ->
         self._transport = None
         output = str(exception) if exception else '\n'.join(self._stderr_lines).rstrip()
         sublime.set_timeout(lambda: self._on_close(self._resolved_command, output, exit_code))
+
+
+class LspDoubleClick(sublime_plugin.TextCommand):
```

Comment:
Still think that `LspOnDoubleClickCommand` would be a more accurate name.

Also the `Command` suffix would be good to have as a convention.